### DISCUSSION TITLE
Validation for orchard core content

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#6902] [Q&A]
I am trying to add validation to the text fields of each content part using javascript but not able to catch the form submit event to do the validation . Anyone guide me on this. how to achieve this.

**Comment by Piedone** [comment]
You can bind your validation logic to the submit event of the form. If you're comfortable with jQuery then [from there too](https://api.jquery.com/submit/), that already exists on the admin.

**Reply by hung-doan** [reply]
@Piedone in case of hooking into the Form Submit Event, how can I do that for a specific content Type ?
Or I have to override the Editor Mode to inject my script ?.

**Reply by Piedone** [reply]
You can target a specific content type's items in a variety of ways, including:

- Include your script from a shape template override targeting that type (see [the docs](https://docs.orchardcore.net/en/dev/docs/reference/modules/Templates/)).
- Include the script based on the URL or even better, route values as explained [here](https://github.com/OrchardCMS/OrchardCore/discussions/6959#discussioncomment-56635).

**Reply by ghost** [reply]
we dont have themes and layout pages to implement the script how can i add validation script to content part any examples that i can refer?

**Reply by Piedone** [reply]
Are you doing the decoupled approach?

**Reply by ghost** [reply]
yes we dont have front end themes

**Reply by Piedone** [reply]
Then it's up to you how to include scripts since Orchard won't do anything on the frontend.
